tests/atlocal.in

```diff
@@ -76,6 +76,7 @@ COBC="${COBC} -fdiagnostics-plain-output"
 COMPILE="${COBC} -x ${FLAGS}"
 COMPILE_ONLY="${COBC} -fsyntax-only ${FLAGS} -Wno-unsupported"
 COMPILE_MODULE="${COBC} -m ${FLAGS}"
+COMPILE_LISTING="${COMPILE_ONLY} -ftversion=V.R.P -fno-ttimestamp -tlines=0"
```

Comment:
please go with
`COMPILE_LISTING="${COMPILE_ONLY} -fttitle=\"GnuCOBOL V.R.P\" -fno-ttimestamp -tlines=0"`
there should be no issues with escaping and while I see the developer ease of `-ftversion` that should not get into the release

Reply:
I dropped the `-ftversion`. However, it is not possible to pass an argument with a space through a command stored in an environment variable in a portable way. So, I modified the behavior of `-fttitle` so that underscores in `-fttitle=<...>` are replaced by spaces.